frontend/lib/features/widget/custom_progress_bar.widget.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:audio_video_progress_bar/audio_video_progress_bar.dart';
```

Comment:
this is a widget that is only used within the audio player. move it to the audio player directory.